### ISSUE TITLE
Copy should be "up-to-date" friendly by default

### ISSUE THREAD

**Issue by davkean** [Microsoft/msbuild#2118]
See this all the time, the default Copy doesn't set SkipUnchangedFiles by default and therefore always runs even if the files haven't changed. This is not obvious. Here's an example of what I needed to set to make my Copy target friendly to MSBuild'isms:

https://github.com/dotnet/project-system/pull/2253

**Comment by KirillOsenkov** [comment]
Yes please, let's set SkipUnchangedFiles to true by default. People who care about this setting will set it explicitly and not rely on the default. But I suspect the vast majority (if not everyone) not setting it aren't doing it on purpose.
